subsys/net/lib/openthread/platform/radio.c

```diff
@@ -220,6 +225,9 @@ static void dataInit(void)
 	net_pkt_append_buffer(tx_pkt, tx_payload);
 
 	sTransmitFrame.mPsdu = tx_payload->data;
+#if defined(CONFIG_IEEE802154_2015)
+        sTransmitFrame.mInfo.mTxInfo.mIeInfo = &sTransmitIeInfo;
```

Comment:
Fix indentation.

Reply:
In general, you can use uncrustify to align the code style with the Zephyr's standard: https://docs.zephyrproject.org/latest/contribute/index.html#uncrustify.